### PR TITLE
Docstring updates for `Axes.fill` and `Axes.pie`

### PR DESCRIPTION
**Pull request by timhoffm**
## PR Summary

As part of #10148: Docstring updates for `Axes.fill` and `Axes.pie`.

### CODE REVIEW

**Comment by timhoffm** [comment]
Ok, I've rewrapped the paragraphs.

File: lib/matplotlib/axes/_axes.py [hunk 1]
```diff
@@ -4695,34 +4696,35 @@ def fill(self, *args, **kwargs):
...
-        args : a variable length argument
-            It allowing for multiple
-            *x*, *y* pairs with an optional color format string; see
-            :func:`~matplotlib.pyplot.plot` for details on the argument
-            parsing.  For example, each of the following is legal::
+        args : sequence of x, y, [color]
+            Each polygon is defined by the lists of *x* and *y* positions of
+            its nodes, optionally followed by by a *color* specifier. See
+            :mod:`matplotlib.colors` for supported color specifiers. The
+            standard color cycle is used for polygons without a color
```
Comment (tacaswell) [inline lib/matplotlib/axes/_axes.py]: iirc this is the same cycle, but a different cycler instance than the one used by `plot`.  Not sure if that is a detail worth including here.

Comment (timhoffm) [reply lib/matplotlib/axes/_axes.py]: Is this important and intentional? Do we want to guarantee this as part of the API?

If not, I would consider it an implementation detail and not document it.

File: lib/matplotlib/axes/_axes.py [hunk 2]
```diff
@@ -2517,30 +2517,32 @@ def pie(self, x, explode=None, labels=None, colors=None,
...
+        resulting pie will have an empty wedge of size ``1 - sum(x)``.
+
+        The wedges are plotted counterclockwise, by default starting from the
+        x-axis.
 
         Parameters
         ----------
         x : array-like
-            The input array used to make the pie chart.
+            The data defining the wedge sizes.
```
Comment (anntzer) [inline lib/matplotlib/axes/_axes.py]: "The wedge sizes." seems enough :)

File: lib/matplotlib/axes/_axes.py [hunk 3]
```diff
@@ -2581,8 +2582,8 @@ def pie(self, x, explode=None, labels=None, colors=None,
             Dict of arguments to pass to the text objects.
```
Comment (anntzer) [inline lib/matplotlib/axes/_axes.py]: just above (in wedgeprops): missing space before double backquote

**Pull request merged**